catalog/CBCBase/CatalogStyle.h

```diff
@@ -16,12 +16,22 @@
 
 #import <UIKit/UIKit.h>
 
-@interface MDCCatalogStyle : NSObject
+/** Shared style that should be used by CatalogByConvention as a base object. */
+@interface CatalogStyle : NSObject
 
+/** Font used in headers: Roboto Mono 16 */
 + (UIFont *)headerFont;
-+ (UIColor *)mdcBlack;
-+ (UIColor *)mdcGrey;
-+ (UIColor *)mdcGreen;
+/** Black (90%) */
++ (UIColor *)primaryColor;
+/** White */
++ (UIColor *)primaryTextColor;
```

Comment:
In the upcoming MDCTheme component, this will be referred to as textColor.  Consider matching that.